pkg/alertmanager/multitenant.go

```diff
@@ -117,6 +119,8 @@ func (cfg *MultitenantAlertmanagerConfig) RegisterFlags(f *flag.FlagSet) {
 
 	f.StringVar(&cfg.ClusterBindAddr, "cluster.listen-address", defaultClusterAddr, "Listen address for cluster.")
 	f.StringVar(&cfg.ClusterAdvertiseAddr, "cluster.advertise-address", "", "Explicit address to advertise in cluster.")
+	f.DurationVar(&cfg.HAGossipInterval, "alertmanager.ha.gossip-interval", cluster.DefaultGossipInterval, "The interval between sending gossip messages. By lowering this value (more frequent) gossip messages are propagated across cluster more quickly at the expense of increased bandwidth usage.")
+	f.DurationVar(&cfg.HAPushPullInterval, "alertmanager.ha.push-pull-interval", cluster.DefaultPushPullInterval, "The interval between gossip state syncs. Setting this interval lower (more frequent) will increase convergence speeds across larger clusters at the expense of increased bandwidth usage.")
```

Comment:
I wouldn't introduce a new prefix `ha` but use `cluster`, which we're already using. My suggestion is to name these new CLI flags (and their YAML field counterpart):

- `alertmanager.cluster-gossip-interval`
- `alertmanager.cluster-push-pull-interval`

Reply:
Oh sorry. Naming is also related to https://github.com/cortexproject/cortex/pull/3677. Let's discuss offline about it.